src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -209,16 +225,25 @@ public String getMessage() {
   }
 
   public JsonApiException toApiException(String format, Object... args) {
+    if (extendError) {
+      return new JsonApiException(this, String.format(format, args));
```

Comment:
Should extract `String.format()` part into helper method since it's called multiple times (to contain logic of formattion wrt `extendError`)

Reply:
not sure I follow, is that to handle errors from String.format () is so +1

Reply:
Just to extract variation in creating `message` (old mode: append title + body; new mode just body) into helper method so it's not repeated in various `toApiException()`s.